CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## [Unreleased]
 
-- No changes yet.
+- Add support for custom WebAssembly plugins for lint and breaking changes plugins. Use the `.wasm` file extension to specify a WebAssembly plugin.
```

Comment:
On line 13, we refer to this as "custom lint and breaking change plugins", however here you refer to them as "lint and breaking changes plugins". The rest of the file also wraps at 100 columns.